src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -944,10 +949,36 @@
         </Style.Triggers>
     </Style>
 
+    <Style
+        x:Key="SZoomFadePreview_Basic"
+        TargetType="{x:Type FrameworkElement}">
+        <Setter
+            Property="Opacity"
+            Value="0.7" />
+        <Style.Triggers>
+            <DataTrigger
+                Binding="{Binding Path=DataContext.Zoom, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:WorkspaceView}}, Converter={StaticResource ZoomToBooleanConverter}}"
+                Value="true">
+                <Setter
+                    Property="Opacity"
+                    Value="0.4" />
+            </DataTrigger>
+        </Style.Triggers>
+    </Style>
+
+    <conv:BooleanToStyleConverter
+        x:Key="SZoomFadePreview"
```

Comment:
Is this used anywhere "SZoomFadePreview" ?

Reply:
no, thanks for pointing this out, I'll take a closer look.

Reply:
to me it looks like this style should have been removed during the UI refresh:
https://github.com/DynamoDS/Dynamo/pull/7069/files
I will remove the converter and the two new styles.